funnel/templates/email_project_registration.html.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "email_layout.html.jinja2" %}
+{% block content -%}
+
+<p>Hi {{ current_auth.user.fullname }},</p>
+
+<p>Your registration for <b>{{ project.title }}</b> was successful.</b></p>
+
+<p>The project is scheduled to begin at {{ project.schedule_start_at_localized|datetime }}</p>
+
+<p>More details about the project are available here: {{ project.url_for(_external=True) }}</p>
```

Comment:
Every sentence in this email needs a `{% trans %}` tag. Further, since this is being sent in a background job, you don't have the user's locale from the current request. The locale must be captured during the request (use `get_locale()` from Baseframe) and passed as a parameter to the background job.